changelog.txt

```diff
@@ -1,3 +1,29 @@
+-------------------------------------------------------------------------------
+CorsixTH 0.69.0 Beta 2 - released June 2025
+-------------------------------------------------------------------------------
+# Features
+* Custom level/campaign creators can now use an existing level file as an
+  overlay to copy and extend their level file with the #overlay variable. See
+  the 'Custom Level' Wiki page for more information.
+
+# Changes
+* You can now hear patients vomiting, instead of just the vomit splatters
+* Lakes no longer show in the Town Map as per the original game
+
+# Translations
+* Russian translation has been updated. Thanks @ARGAMX
+
+# Bug Fixes
+* Fixed a crash in General Diagnosis
+* Fixed a compatibility issue with mood markers on old saves
+* Fixed a crash with the grim reaper
+* Fixed a bug with the hotkey assignment menu
+* Fixed some anomalies in the original campaign's map files
+* Fixed an issue with handymen ignoring cleaning duties. Please note existing
```

Comment:
This was a regression in beta 1 right?

I'd like to track regressions so we can write them out of the final release notes.

Reply:
I think most bug fixes are regressive behaviours? Do you have a preference on how to mark them as such or are you fine as "Fixed regression"?

Reply:
Yes, fixed 0.69-beta1 regression:

Reply:
All but one of those bug fixes was a regression. It looked silly adding shared text to all of them so it has its own header instead.